cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -271,11 +279,19 @@ message CassandraServerRunTask {
     required string version = 1;
     repeated string command = 2;
     optional TaskEnv taskEnv = 3;
-    repeated TaskFile taskFiles = 4;
-    required TaskConfig taskConfig = 5;
-    required JmxConnect jmx = 6;
+    required CassandraServerConfig cassandraServerConfig = 4;
+    required JmxConnect jmx = 5;
+}
+
+message UpdateConfigTask {
+    required CassandraServerConfig cassandraServerConfig = 1;
+}
+
+message CassandraServerConfig {
```

Comment:
Should the `TaskEnv` object also be moved into here? It contains config changes for cassandra-env.sh.